mocca-client/src/test/java/com/paypal/mocca/client/sample/SampleClient.java

```diff
@@ -23,6 +23,9 @@
     @Query
     SampleResponseDTO getOneSample(@Var("sampleRequest") SampleRequestDTO sampleRequestDTO);
 
+    @Query
+    SampleResponseDTO getOneValidSample(@Var("sampleRequest") ValidatedRequestDTO validatedRequestDTO);
```

Comment:
Could we add `@NotNull` to `validatedRequestDTO` and add some tests making sure that is enforced?

Reply:
sure

Reply:
Actually now that I think about it, this won't work I think. The code currently only validates the object itself not the method arguments. I think this is doable but it would take some time.

Reply:
Ok. Let's do it please. This is an important requirement.